app/models/waste_exemptions_engine/registration.rb

```diff
@@ -18,7 +16,7 @@ class Registration < ::WasteExemptionsEngine::ApplicationRecord
 
     scope :renewals, -> { where.not(referring_registration_id: nil) }
 
-    scope :contact_email_is_not_nccc, -> { where.not(contact_email: NCCC_EMAIL) }
+    scope :contact_email_is_not_blank, -> { where.not(contact_email: nil) }
```

Comment:
"blank" and "nil" are not the same - maybe rename this to `contact_email_present` and also check for empty string values?